github/data_source_github_ip_ranges.go

```diff
@@ -22,7 +22,12 @@ func dataSourceGithubIpRanges() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
-			"pages": {
```

Comment:
This pages attribute disappears, but is still present in the docs file `ip_ranges.html.markdown`. Did you mean to remove it?

Reply:
No you're quite right, good spot. I will make sure this is added back in!